Extensions/shuffle_queue.js

```diff
@@ -29,92 +29,132 @@ XKit.extensions.shuffle_queue = new Object({
+		$("#xshrinkposts_button").click(function() {
+			if (XKit.installed.is_running("shorten_posts")) {
+				XKit.window.show(
+					"Unable to turn on Shrink Posts",
+					"Using the Shrink Posts option and Shorten Posts together creates a small mess that no one really wants to see. " +
+					"If you still want to use the Shrink Posts functionality of Enhanced Queue, disable the Shorten Posts extension first.",
+
+					"error",
+
+					'<div class="xkit-button default" id="xkit-close-message">OK</div>'
+				);
+			} else {
+				let $button = $(this).toggleClass("xkit-queue-option-button-on");
+				if ($button.hasClass("xkit-queue-option-button-on")) {
+					$button.find(".count").html("on");
+					XKit.storage.set("shuffle_queue", "shrink_posts", "true");
+					XKit.tools.add_css(
+						".post_header { display: none; }" +
+						".post .post_content_inner, .post .post_media { height: 70px !important; overflow: hidden !important; }" +
+						".post .post_content { pointer-events: none !important; height: 70px !important; overflow: hidden !important; border: 1px dashed rgb(200,200,200); }",
+
+						"shuffle_queue_mini_posts"
+					);
 				} else {
-
-					$(this).find(".count").html("on");
-
-					XKit.storage.set("shuffle_queue", "hide_options", "false");
-
-					XKit.tools.remove_css("shuffle_queue_hide_options");
-
+					$button.find(".count").html("off");
+					XKit.storage.set("shuffle_queue", "shrink_posts", "false");
+					XKit.tools.remove_css("shuffle_queue_mini_posts");
 				}
+			}
+		});
+		$("#xqueueoptions_button").click(function() {
+			let $button = $(this).toggleClass("xkit-queue-option-button-on");
+
+			if ($button.hasClass("xkit-queue-option-button-on")) {
+				$button.find(".count").html("off");
+				XKit.storage.set("shuffle_queue", "hide_options", "true");
+				XKit.tools.add_css(" .dashboard_options_form { display: none; }", "shuffle_queue_hide_options");
+			} else {
+				$button.find(".count").html("on");
+				XKit.storage.set("shuffle_queue", "hide_options", "false");
+				XKit.tools.remove_css("shuffle_queue_hide_options");
+			}
+		});
 
-				return false;
-			});
-
-			$("#xshrinkposts_button").click(function() {
-				//Check if Shorten Posts is enabled
-				if (!XKit.installed.is_running("shorten_posts")) {
-					//Shorten Posts is not enabled, no issues there
-					$(this).toggleClass("xkit-queue-option-button-on");
-
-					if ($(this).hasClass("xkit-queue-option-button-on")) {
-
-						$(this).find(".count").html("on");
-
-						XKit.storage.set("shuffle_queue", "shrink_posts", "true");
-
-						XKit.tools.add_css(" .post_header { display: none; }  .post .post_content_inner, .post .post_media { height: 70px !important; overflow: hidden !important; } .post .post_content { pointer-events: none !important; height: 70px !important; overflow: hidden !important; border: 1px dashed rgb(200,200,200); } ", "shuffle_queue_mini_posts");
+		var shrink_posts = XKit.storage.get("shuffle_queue", "shrink_posts", "false");
```

Comment:
If you want to be fancy these pref vars could be const but I respect wanting to avoid the churn of let/consting everything

Reply:
yeah it's kinda worth the churn for nice block-level scoping but when its function-scoped either way i would rather just leave it alone unless its like, actually important that a value is const and/or not obvious from the line or two below its declaration